--- origsrc/gcompris-8.4.8/configure.in	2009-01-11 23:15:32.718375000 -0600
+++ src/gcompris-8.4.8/configure.in	2009-01-11 23:34:18.937125000 -0600
@@ -294,7 +294,7 @@ dnl
 dnl === DIRECTORY DEFINITIONS ===
 dnl
 
-if test "x$platform_win32" = "xno" ; then
+if test "x$native_win32" = "xno" ; then
 
   dnl Set PACKAGE_LOCALE_DIR in config.h.
   AC_DEFINE_UNQUOTED(PACKAGE_LOCALE_DIR, "/usr/share/locale", [GCompris locale directory])
--- origsrc/gcompris-8.4.8/src/boards/py-gcompris-board.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.8/src/boards/py-gcompris-board.c	2009-01-11 23:34:18.952750000 -0600
@@ -172,7 +172,7 @@ pyGcomprisBoardType_setattr(pyGcomprisBo
 }
 
 static PyTypeObject pyGcomprisBoardType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.8/src/boards/py-gcompris-class.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.8/src/boards/py-gcompris-class.c	2009-01-11 23:34:18.952750000 -0600
@@ -96,7 +96,7 @@ pyGcomprisClassType_setattr(pyGcomprisCl
 }
 
 static PyTypeObject pyGcomprisClassType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.8/src/boards/py-gcompris-group.c	2007-08-30 17:52:28.000000000 -0500
+++ src/gcompris-8.4.8/src/boards/py-gcompris-group.c	2009-01-11 23:34:18.968375000 -0600
@@ -85,7 +85,7 @@ pyGcomprisGroupType_setattr(pyGcomprisGr
 }
 
 static PyTypeObject pyGcomprisGroupType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.8/src/boards/py-gcompris-profile.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.8/src/boards/py-gcompris-profile.c	2009-01-11 23:34:18.968375000 -0600
@@ -100,7 +100,7 @@ pyGcomprisProfileType_setattr(pyGcompris
 }
 
 static PyTypeObject pyGcomprisProfileType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.8/src/boards/py-gcompris-properties.c	2007-08-30 17:52:28.000000000 -0500
+++ src/gcompris-8.4.8/src/boards/py-gcompris-properties.c	2009-01-11 23:34:18.984000000 -0600
@@ -186,7 +186,7 @@ pyGcomprisPropertiesType_setattr(pyGcomp
 }
 
 static PyTypeObject pyGcomprisPropertiesType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.8/src/boards/py-gcompris-user.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.8/src/boards/py-gcompris-user.c	2009-01-11 23:34:18.984000000 -0600
@@ -79,7 +79,7 @@ pyGcomprisUserType_setattr(pyGcomprisUse
 }
 
 static PyTypeObject pyGcomprisUserType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.8/src/boards/py-gcompris-wordlist.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.8/src/boards/py-gcompris-wordlist.c	2009-01-11 23:34:18.999625000 -0600
@@ -91,7 +91,7 @@ pyGcomprisWordlistType_setattr(pyGcompri
 }
 
 static PyTypeObject pyGcomprisWordlistType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.8/src/boards/py-mod-anim.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.8/src/boards/py-mod-anim.c	2009-01-11 23:34:18.999625000 -0600
@@ -39,7 +39,7 @@ static PyMethodDef AnimationMethods[] = 
 };
 
 static PyTypeObject py_GcomprisAnimationType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
@@ -85,7 +85,7 @@ static PyTypeObject py_GcomprisAnimation
 };
 
 static PyTypeObject py_GcomprisAnimCanvasType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
