--- origsrc/gcompris-8.4.12/configure.in	2009-04-23 13:13:03.233750000 -0500
+++ src/gcompris-8.4.12/configure.in	2009-04-23 14:00:23.546250000 -0500
@@ -227,9 +227,9 @@ AC_ARG_ENABLE(debug,
           USE_DEBUG="$enableval", USE_DEBUG="no")
 if test "x$USE_DEBUG" == "xyes" ; then
     AC_DEFINE_UNQUOTED(DEBUG, 1, "Enable debug messages.")
-    CFLAGS="$CFLAGS $OSX_CFLAGS -Wall -Werror -g"
+    CFLAGS="$CFLAGS $OSX_CFLAGS -Wall -g"
 else
-    CFLAGS="$CFLAGS $OSX_CFLAGS -Wall -Werror"
+    CFLAGS="$CFLAGS $OSX_CFLAGS -Wall"
 fi
 
 dnl WIN32 Specifics
