--- origsrc/gcompris-9.5/src/gcompris/Makefile.am	2011-01-08 21:28:16.375162700 -0600
+++ src/gcompris-9.5/src/gcompris/Makefile.am	2011-01-08 21:31:28.364697500 -0600
@@ -118,7 +118,7 @@ gcompris_LDFLAGS += \
 endif
 
 gcompris_LDADD = \
-	$(top_builddir)/src/goocanvas/src/libgoocanvas.la \
+	$(top_builddir)/src/goocanvas/src/libgcompriscanvas.la \
 	$(GCOMPRIS_LIBS) $(sqlite_ldadd) $(XML_LIBS) $(GNET_LIBS) \
 	$(DBUS_LIBS) \
 	$(AUDIO_LIBS) \
--- origsrc/gcompris-9.5/src/goocanvas/src/Makefile.am	2010-10-24 18:03:41.000000000 -0500
+++ src/gcompris-9.5/src/goocanvas/src/Makefile.am	2011-01-08 21:32:43.978029300 -0600
@@ -1,22 +1,22 @@
 ## Process this file with automake to produce Makefile.in
 
 INCLUDES = \
-	-DG_LOG_DOMAIN=\"GooCanvas\" \
+	-DG_LOG_DOMAIN=\"GComprisCanvas\" \
 	@GCOMPRIS_CFLAGS@
 
 libdir = $(plugindir)
-lib_LTLIBRARIES = libgoocanvas.la
+lib_LTLIBRARIES = libgcompriscanvas.la
 
-libgoocanvas_la_LIBADD = @GCOMPRIS_LIBS@ $(INTLLIBS)
+libgcompriscanvas_la_LIBADD = @GCOMPRIS_LIBS@ $(INTLLIBS)
 
 if PLATFORM_WIN32
 no_undefined = -no-undefined
 endif
 
 libtool_opts =				\
-  -rpath $(libdir)
+  -rpath $(libdir) $(no_undefined)
 
-libgoocanvas_la_LDFLAGS = $(libtool_opts)
+libgcompriscanvas_la_LDFLAGS = $(libtool_opts)
 
 libgoocanvasincludedir = $(includedir)/goocanvas-1.0
 
@@ -41,7 +41,7 @@ libgoocanvas_public_headers =		\
 	goocanvas.h			\
 	goocanvassvg.h
 
-libgoocanvas_la_SOURCES =		\
+libgcompriscanvas_la_SOURCES =		\
 	goocanvasatk.h			\
 	goocanvasatk.c			\
 	goocanvasellipse.c		\
