--- origsrc/gcompris-9.5/configure.ac	2011-01-08 21:28:16.765163400 -0600
+++ src/gcompris-9.5/configure.ac	2011-01-08 21:31:08.022262000 -0600
@@ -205,7 +205,7 @@ if test "x$USE_DEBUG" == "xyes" ; then
     AC_DEFINE_UNQUOTED(DEBUG, 1, "Enable debug messages.")
     CFLAGS="$CFLAGS $OSX_CFLAGS -Wall -Werror -g"
 else
-    CFLAGS="$CFLAGS $OSX_CFLAGS -Wall -Werror -O2"
+    CFLAGS="$CFLAGS $OSX_CFLAGS -Wall -O2"
 fi
 
 dnl disable activation code
