--- origsrc/gcompris-8.4.5/configure.in	2008-05-17 23:45:49.562500000 -0500
+++ src/gcompris-8.4.5/configure.in	2008-05-18 00:52:22.281250000 -0500
@@ -11,6 +11,7 @@
 AC_ISC_POSIX
 
 GTK_REQUIRED=2.4.0
+GNOME_CANVAS_REQUIRED=2.16.0
 
 dnl
 dnl Start of pkg-config checks
@@ -20,6 +21,7 @@
 
 PKG_CHECK_MODULES(GCOMPRIS, dnl
  [gtk+-2.0 >= $GTK_REQUIRED dnl
+  libgnomecanvas-2.0 >= $GNOME_CANVAS_REQUIRED dnl
   gstreamer-0.10])
 
 AC_SUBST(GCOMPRIS_CFLAGS)
@@ -294,7 +296,7 @@
 dnl === DIRECTORY DEFINITIONS ===
 dnl
 
-if test "x$platform_win32" = "xno" ; then
+if test "x$native_win32" = "xno" ; then
 
   dnl Set PACKAGE_LOCALE_DIR in config.h.
   AC_DEFINE_UNQUOTED(PACKAGE_LOCALE_DIR, "/usr/share/locale", [GCompris locale directory])
--- origsrc/gcompris-8.4.5/docs/eu/Makefile.am	2008-05-17 23:45:49.593750000 -0500
+++ src/gcompris-8.4.5/docs/eu/Makefile.am	2008-05-18 15:27:03.906250000 -0500
@@ -15,9 +15,9 @@
 # FIXME: Is it possible to have I18N info files installed ?
 #info_TEXINFOS = gcompris.texi
 
-install-data-local:
+install-data-local: gcompris.html
 	$(mkinstalldirs) $(DESTDIR)$(gcompris_helpdir)
-	$(INSTALL_DATA)  gcompris.html $(DESTDIR)$(gcompris_helpdir)
+	$(INSTALL_DATA)  $< $(DESTDIR)$(gcompris_helpdir)
 	$(INSTALL_DATA)  $(srcdir)/overview.jpg $(DESTDIR)$(gcompris_helpdir)
 	$(INSTALL_DATA)  $(srcdir)/topic.dat $(DESTDIR)$(gcompris_helpdir)
 
--- origsrc/gcompris-8.4.5/docs/fr/Makefile.am	2008-05-17 23:45:49.593750000 -0500
+++ src/gcompris-8.4.5/docs/fr/Makefile.am	2008-05-18 15:26:21.718750000 -0500
@@ -15,9 +15,9 @@
 # FIXME: Is it possible to have I18N info files installed ?
 #info_TEXINFOS = gcompris.texi
 
-install-data-local:
+install-data-local: gcompris.html
 	$(mkinstalldirs) $(DESTDIR)$(gcompris_helpdir)
-	$(INSTALL_DATA)  gcompris.html $(DESTDIR)$(gcompris_helpdir)
+	$(INSTALL_DATA)  $< $(DESTDIR)$(gcompris_helpdir)
 	$(INSTALL_DATA)  $(srcdir)/overview.jpg $(DESTDIR)$(gcompris_helpdir)
 	$(INSTALL_DATA)  $(srcdir)/topic.dat $(DESTDIR)$(gcompris_helpdir)
 
--- origsrc/gcompris-8.4.5/src/Makefile.am	2007-05-22 15:15:05.000000000 -0500
+++ src/gcompris-8.4.5/src/Makefile.am	2008-05-18 00:48:52.953125000 -0500
@@ -1,3 +1,3 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = libart_lgpl libgnomecanvas gcompris boards
+SUBDIRS = gcompris boards
--- origsrc/gcompris-8.4.5/src/boards/Makefile.am	2008-04-12 16:54:02.000000000 -0500
+++ src/gcompris-8.4.5/src/boards/Makefile.am	2008-05-18 12:07:27.234375000 -0500
@@ -19,7 +19,7 @@
 if PLATFORM_WIN32
 shared = -shared
 no_undefined = -no-undefined
-gc_libs = $(GCOMPRIS_LIBS)
+gc_libs = $(GCOMPRIS_LIBS) -Wl,$(top_builddir)/src/gcompris/libgcompris.a
 dll_export = -DBOARD_DLL_EXPORT
 else
 shared =
@@ -77,13 +77,14 @@
 
 libdir = $(plugindir)
 INCLUDES = \
-	-I$(top_builddir)/src \
-	-I$(top_builddir)/intl \
 	$(GCOMPRIS_CFLAGS) \
 	$(XML_CFLAGS) \
 	$(PYTHON_CFLAGS) \
 	$(PYGTK_CFLAGS) \
 	$(sqlite_cflags) \
+	-I$(top_builddir)/src \
+	-I$(top_srcdir)/src \
+	-I$(top_builddir)/intl \
 	$(dll_export)
 
 libawele_la_LDFLAGS = $(shared) $(no_undefined) -module -avoid-version $(gc_libs)
--- origsrc/gcompris-8.4.5/src/boards/py-gcompris-board.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.5/src/boards/py-gcompris-board.c	2008-05-18 15:10:49.031250000 -0500
@@ -172,7 +172,7 @@
 }
 
 static PyTypeObject pyGcomprisBoardType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.5/src/boards/py-gcompris-class.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.5/src/boards/py-gcompris-class.c	2008-05-18 15:11:13.734375000 -0500
@@ -96,7 +96,7 @@
 }
 
 static PyTypeObject pyGcomprisClassType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.5/src/boards/py-gcompris-group.c	2007-08-30 17:52:28.000000000 -0500
+++ src/gcompris-8.4.5/src/boards/py-gcompris-group.c	2008-05-18 15:11:24.000000000 -0500
@@ -85,7 +85,7 @@
 }
 
 static PyTypeObject pyGcomprisGroupType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.5/src/boards/py-gcompris-profile.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.5/src/boards/py-gcompris-profile.c	2008-05-18 15:11:27.109375000 -0500
@@ -100,7 +100,7 @@
 }
 
 static PyTypeObject pyGcomprisProfileType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.5/src/boards/py-gcompris-properties.c	2007-08-30 17:52:28.000000000 -0500
+++ src/gcompris-8.4.5/src/boards/py-gcompris-properties.c	2008-05-18 15:11:29.765625000 -0500
@@ -186,7 +186,7 @@
 }
 
 static PyTypeObject pyGcomprisPropertiesType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.5/src/boards/py-gcompris-user.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.5/src/boards/py-gcompris-user.c	2008-05-18 15:11:33.187500000 -0500
@@ -79,7 +79,7 @@
 }
 
 static PyTypeObject pyGcomprisUserType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.5/src/boards/py-gcompris-wordlist.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.5/src/boards/py-gcompris-wordlist.c	2008-05-18 15:11:39.562500000 -0500
@@ -91,7 +91,7 @@
 }
 
 static PyTypeObject pyGcomprisWordlistType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.5/src/boards/py-mod-anim.c	2007-08-30 17:52:27.000000000 -0500
+++ src/gcompris-8.4.5/src/boards/py-mod-anim.c	2008-05-18 15:13:13.375000000 -0500
@@ -39,7 +39,7 @@
 };
 
 static PyTypeObject py_GcomprisAnimationType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
@@ -85,7 +85,7 @@
 };
 
 static PyTypeObject py_GcomprisAnimCanvasType = {
-#if defined(WIN32)
+#if defined(G_PLATFORM_WIN32)
   PyObject_HEAD_INIT(NULL)
 #else /* ! WIN32 */
   PyObject_HEAD_INIT(&PyType_Type)
--- origsrc/gcompris-8.4.5/src/gcompris/Makefile.am	2008-02-17 17:37:13.000000000 -0600
+++ src/gcompris-8.4.5/src/gcompris/Makefile.am	2008-05-18 12:06:37.375000000 -0500
@@ -30,9 +30,11 @@
 if PLATFORM_WIN32
 no_undefined = -no-undefined
 shared = -shared
+export_symbols = -Wl,--export-all-symbols -Wl,--out-implib,libgcompris.a
 else
 no_undefined =
 shared =
+export_symbols = -export-dynamic
 endif
 
 if WITH_NSBUNDLE
@@ -119,9 +121,7 @@
 	wordlist.c \
 	wordlist.h
 
-gcompris_LDFLAGS = \
-	-export-dynamic $(top_builddir)/src/libgnomecanvas/libgnomecanvas-2.a \
-	$(top_builddir)/src/libart_lgpl/libart_lgpl_2.a
+gcompris_LDFLAGS = $(export_symbols)
 
 
 gcompris_LDADD = \
